lib/taskana-core-test/src/test/java/acceptance/user/UserServiceAccTest.java

```diff
@@ -1124,6 +1361,27 @@ void should_ReturnEmptyDomains_When_PropertyIsNotSetAndGroupHasPermission() thro
 
         assertThat(userInDatabase.getDomains()).isEmpty();
       }
+
+      @WithAccessId(user = "user-1-1")
+      @Test
+      void should_ReturnEmptyDomains_When_PropertyIsNotSetAndPermissionHasPermission()
```

Comment:
What do you mean by PermissionHasPermission? Could you adjust the test name so that it's easier to understand?

Reply:
fixed